pkg/maintenance/alertmanagerMaintenance.go

```diff
@@ -138,31 +138,36 @@ func (amm *alertManagerMaintenance) StartControlPlane(endsAt time.Time, version
 
 // Start a worker node maintenance in Alertmanager for version
 // Time is converted to UTC
-func (amm *alertManagerMaintenance) SetWorker(endsAt time.Time, version string) error {
+func (amm *alertManagerMaintenance) SetWorker(endsAt time.Time, version string, count int32) error {
```

Comment:
Does the same logic work if you compare the endAt time instead of passing the count?

Reply:
We pass the `endAt` as a dynamic value "time.Now().Add(xxx)" in the reconcile loop. And we cannot store the value in anywhere.
I do not think it is possible to find the exact silence by the `endAt`

Reply:
Gotcha :+1: